activesupport/lib/active_support/string_inquirer.rb

```diff
@@ -9,6 +9,34 @@ module ActiveSupport
   #
   #   Rails.env.production?
   class StringInquirer < String
+
+    def initialize(value, valid_values = [])
```

Comment:
> Isn't it a concern?

It might very well be. But we need a benchmark to see any difference in practice :smile:

We also need to test instantiation times, since the methods are defined up front. My guess is that cost is probably way higher than just routing to `method_missing` :smile: